editor/src/components/canvas/canvas-strategies/strategies/keyboard-absolute-move-strategy.ts

```diff
@@ -44,6 +44,16 @@ export function keyboardAbsoluteMoveStrategy(
 ): CanvasStrategy | null {
   const { pathsWereReplaced, paths: selectedElements } =
     retargetStrategyToChildrenOfFragmentLikeElements(canvasState)
+
+  if (
+    pathsWereReplaced &&
+    selectedElements.some((originalTarget) =>
+      MetadataUtils.isComponentInstanceFromMetadata(canvasState.startingMetadata, originalTarget),
+    )
+  ) {
```

Comment:
is there any strategy left that uses `retargetStrategyToChildrenOfFragmentLikeElements` and _doesn't_ bail out on isComponentInstanceFromMetadata? because if not, then I think we should move the component check inside!

not sure about the fallout, but it very much looks like `getElementFragmentLikeType` can be changed so it only returns `'sizeless-div'` as a result if `isComponentInstanceFromMetadata` is false.

Reply:
There's other utility functions which also invoke `retargetStrategyToChildrenOfFragmentLikeElements` which is why I left the logic like this.

I think if we want to refactor `getElementFragmentLikeType` it should be done in a separate piece of work, so that this doesn't become a forever PR.